src/locales/en/common.json

```diff
@@ -139,6 +149,7 @@
   "navbar": {
     "consumer": "Consumer",
     "pluginGlobalRules": "Plugin Global Rules",
+    "protos": "Protos",
```

Comment:
Why is this the only one `key` expressed in the plural?

Reply:
That's because others are wrong. They will be fixed in their own PR.